### PR TITLE
Careers why elewa

### PR DESCRIPTION
**Pull request by Eddahkarambu**
# Description

Created the  Careers why Elewa

- to achieve this we had to :

- Create the careers -value- proposition component
- Add the HTML for the component
- Style the component

# Desktop preview
why-elewa page

# Phone preview
elewa why 2
# Checklist
[✅] My code follows the style guidelines of this project
[✅] I have performed a self-review of my code
[✅] I have commented my code, particularly in hard-to-understand areas
[✅] I have made corresponding changes to the documentation
[✅] My changes generate no new warnings
[✅] I have added tests that prove my fix is effective or that my feature works
[✅] New and existing unit tests pass locally with my changes
[✅] Any dependent changes have been merged and published in downstream modules

This code was written by @Esther uge and I

### CODE REVIEW

**Comment by Eddahkarambu** [comment]
@IanOdhiambo9  Kindly have a look at this pull request.
I was in the middle of doing it before the issue  was merged

File: libs/pages/elewa/careers/src/lib/components/careers-value-proposition/careers-value-proposition.component.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<div id="careers-why">
+  <div class="img-container">
+    <img src="{{ imageUrl1 }}" class="img1" />
```
Comment (IanOdhiambo9) [inline libs/pages/elewa/careers/src/lib/components/careers-value-proposition/careers-value-proposition.component.html:3]: we prefer to use  instead

File: libs/pages/elewa/careers/src/lib/components/careers-value-proposition/careers-value-proposition.component.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    <div class="container">
+      <h1 id="title">{{ title }}</h1>
+      <p id="paragraph">
+        {{ paragraph }}
+      </p>
+    </div>
+  </div>
+  <img src="{{ imageUrl2 }}" class="img2" />
+  <!-- <div class="container"> -->
+  <!-- </div> -->
```
Comment (IanOdhiambo9) [inline libs/pages/elewa/careers/src/lib/components/careers-value-proposition/careers-value-proposition.component.html:13]: always clean up. remove code that is not being used

File: libs/pages/elewa/careers/src/lib/components/careers-value-proposition/careers-value-proposition.component.scss [hunk 3]
```diff
@@ -0,0 +1,95 @@
+#careers-why {
+  width: 100%;
+  height: auto;
+  //   border: solid;
```
Comment (IanOdhiambo9) [inline libs/pages/elewa/careers/src/lib/components/careers-value-proposition/careers-value-proposition.component.scss:4]: same

File: libs/pages/elewa/careers/src/lib/components/careers-value-proposition/careers-value-proposition.component.scss [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+  display: flex;
+  //   flex-direction: column;
```
Comment (IanOdhiambo9) [inline libs/pages/elewa/careers/src/lib/components/careers-value-proposition/careers-value-proposition.component.scss:6]: same

File: libs/pages/elewa/careers/src/lib/components/careers-value-proposition/careers-value-proposition.component.scss [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+  height: 400px;
+  align-self: flex-start;
+}
+.img2 {
+  width: 400px;
+  height: 500px;
+}
+.container {
+    width: 100%;
+  //   margin-left: 11rem;
```
Comment (IanOdhiambo9) [inline libs/pages/elewa/careers/src/lib/components/careers-value-proposition/careers-value-proposition.component.scss:39]: same

File: libs/pages/elewa/careers/src/lib/components/careers-value-proposition/careers-value-proposition.component.scss [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+//       width: 90%;
+//     }
+
+//     .img-container img:last-child {
+//       margin-top: 170%;
+//       margin-left: 10%;
+//       padding-top: 10%;
+//     }
+//   }
+// }
```
Comment (IanOdhiambo9) [inline libs/pages/elewa/careers/src/lib/components/careers-value-proposition/careers-value-proposition.component.scss:95]: too much dead code

**Review by IanOdhiambo9 — COMMENTED** [review]
The issue was well done. Just a few clean up issues

**Pull request closed without merging**